traitsui/tests/test_animatedGIF_editor.py

```diff
@@ -0,0 +1,48 @@
+#  Copyright (c) 2005-2020, Enthought, Inc.
+#  All rights reserved.
+#
+#  This software is provided without warranty under the terms of the BSD
+#  license included in LICENSE.txt and may be redistributed only
+#  under the conditions described in the aforementioned license.  The license
+#  is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+#  Thanks for using Enthought open source!
+#
+""" Tests to exercise logic for layouts, e.g. VGroup, HGroup.
+"""
+
+import unittest
+
+from traits.api import HasTraits, File, Bool
+from traitsui.api import View, Item
+from traitsui.wx.animated_gif_editor import AnimatedGIFEditor
+import pkg_resources
+
+from traitsui.tests._tools import (
+    create_ui,
+    skip_if_not_wx,
+)
```

Comment:
Nit-picky: 
The PEP-8 guide _recommends_ the following::

> Imports should be grouped in the following order:
> Standard library imports.
> Related third party imports.
> Local application/library specific imports.
> You should put a blank line between each group of imports.

For ETS, we introduce another level of groupings (https://github.com/enthought/ets/blob/master/docs/source/eeps/eep-1.rst#imports)

> Standard library imports.
> Third-party imports (comtraits, PyOpenGL).
> Enthought imports (traits, enaml, chaco, etc.).
> Local imports.

The next level of recommendation is alphabetically ordering the imports (that is my weakness really, I can't order letters correctly).
All of these guidelines are a bit dull, but it is not merely aesthetic. When adhered to, it helps minimizing trivial merge conflicts. As developers agree to the same convention consistently, it also helps quickly identifying dependencies in the import blocks. While it does not make a world of differences, it is something that can be agreed to once and for all (more efficient in the long run).
```suggestion
import unittest

import pkg_resources

from traits.api import Bool, File, HasTraits,
from traitsui.api import Item, View
from traitsui.tests._tools import (
    create_ui,
    skip_if_not_wx,
)
```

(I removed the import from `traitsui.wx`, see comment below).